### PR TITLE
Adding phpunit test for Validators

### PR DESCRIPTION
**Pull request by wiewiurdp**

### CODE REVIEW

File: module/attribute/tests/Application/Form/Event/AttributeFormSubscriberTest.php [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+                'constraints' => [
+                    new NotBlank(['message' => 'At least one image format is required']),
+                ],
+            ]
+        );
+        $subscriber = new AttributeFormSubscriber($this->event);
+        $subscriber->onPreSubmit($this->event);
+    }
+
+
```
Comment (rprzedzik) [inline module/attribute/tests/Application/Form/Event/AttributeFormSubscriberTest.php]: Unnecesery lines

File: module/category/tests/Infrastructure/Validator/CategoryCodeValidatorTest.php [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+    /**
+     * @expectedException \Symfony\Component\Validator\Exception\ValidatorException
+     */
+    public function testWrongValueProvided(): void
+    {
+        $this->validator->validate(new \stdClass(), new CategoryCode());
+    }
+
+
```
Comment (rprzedzik) [inline module/category/tests/Infrastructure/Validator/CategoryCodeValidatorTest.php]: Unnecesery line

File: module/product-simple/tests/Infrastructure/Action/Extension/ProductAttributeExtensionTest.php [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+    {
+        $this->record->expects($this->once())->method('has')->willReturn(true);
+        $this->record->expects($this->once())->method('getColumns')->willReturn(['color' => new StringValue('black')]);
+        $this->query->expects($this->any())->method('findAttributeType')->willReturn(new AttributeType('SELECT'));
+        $this->messageBus->expects($this->once())->method('dispatch')->willReturn(new Envelope($this->createMock(AttributeType::class)));
+        $result = $this->extension->extend($this->record, $this->data);
+        $this->assertEquals('black', $result['attributes']['color']->getValue());
+    }
+
+
```
Comment (rprzedzik) [inline module/product-simple/tests/Infrastructure/Action/Extension/ProductAttributeExtensionTest.php]: Unnecesery line

File: phpunit.xml.dist [hunk 4]
```diff
@@ -29,10 +29,15 @@
...
-            <directory>./module/*/src/</directory>
+            <directory>./module/*/src/Application</directory>
+            <directory>./module/*/src/Domain</directory>
+            <directory>./module/*/src/Infrastructure</directory>
             <directory>./src/</directory>
             <exclude>
                 <directory suffix="Kernel.php">./src</directory>
+                <directory>./module/*/src/Application/Controller</directory>
+                <directory>./module/*/src/Application/DependencyInjection</directory>
+
```
Comment (BastekBielawski) [inline phpunit.xml.dist]: Remove empty line

File: module/translation-deepl/tests/Infrastructure/Validator/DeeplLanguageValidatorTest.php [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+ */
+class DeeplLanguageValidatorTest extends ConstraintValidatorTestCase
+{
+
+    protected function setUp()
+    {
+        parent::setUp();
+    }
+
+
```
Comment (rprzedzik) [inline module/translation-deepl/tests/Infrastructure/Validator/DeeplLanguageValidatorTest.php]: Unnecesery line

Comment (BastekBielawski) [inline module/translation-deepl/tests/Infrastructure/Validator/DeeplLanguageValidatorTest.php]: Remove setUp method

File: phpunit.xml.dist (hunk 4 above)
Comment (rprzedzik) [inline phpunit.xml.dist]: Unnecesery line

**Pull request merged**